examples/erc1155/tests/erc1155.rs

```diff
@@ -52,6 +52,10 @@ async fn constructs(alice: Account) -> eyre::Result<()> {
     let uri = contract.uri(U256::from(1)).call().await?.uri;
     assert_eq!(URI, uri);
 
+    let paused = contract.paused().call().await?.paused;
+
+    assert_eq!(false, paused);
```

Comment:
```suggestion
    assert!(!paused);
```
nit: this is a general pattern throughout our codebase that could be shortened. Unless you prefer the explicit equality check?